ape_arbitrum/ecosystem.py

```diff
@@ -40,6 +40,8 @@ class ApeArbitrumError(ApeException):
 
 
 class ArbitrumReceipt(Receipt):
+    gas_used_for_L1: HexInt
```

Comment:
This will help it serialize back to the expected key:

```suggestion
    gas_used_for_L1: HexInt = Field(alias="gasUsedForL1")
```

Reply:
had to add a default value for mypy